package.json

```diff
@@ -91,8 +91,8 @@
   "xo": {
     "esnext": true,
     "ignores": [
-      "test/test.js",
-      "dist/**"
+      "dist/**",
+      "lib/index.js"
```

Comment:
The last comment in this issue https://github.com/babel/babel-eslint/issues/78#issuecomment-126779546 indicates it still occurs even with that plugin.